src/frontend/utils/farger.ts

```diff
@@ -2,3 +2,4 @@
  * Pga att farger kun finnes tilgjengelige i less finnes noen her også
  */
 export const navLillaLighten20 = '#826BA1';
+export const navGra60 = '#78706A';
```

Comment:
Har eksportert navfarger slik for at vi skal kunne bruke dem med Styled-components.
De vanlige variablene funker bare med less-filer

Reply:
Jeg flyttet disse til en fil som allerede fantes, styles.ts
65ce740